blueprint/src/common.tex

```diff
@@ -0,0 +1,8 @@
+\title{Algebra in Lean}
+\author{Will Harris, Justin Morrill, Dominic King, Adam Kern, Clara Henne, Arim Lim, 
+William Ho, Stavan Jain, Anoushka Sinha, Ricardo Prado Cunha}
```

Comment:
I think you should use `\and` separating the list of authors, rather than commas. See https://texdoc.org/serve/latex2e/0 page 174. Also, trailing whitespace after "Arim Lim,".